From 181af945894b455f2bdab021a6090f699b9d40df Mon Sep 17 00:00:00 2001
From: Emmanuel Blot <emmanuel.blot@free.fr>
Date: Fri, 14 Aug 2026 17:19:14 +0200
Subject: [PATCH] hw/sensor: tmp421: describe the temperature properties

Attach a description to each temperature property so it shows up in
qom-list-properties.

Signed-off-by: Emmanuel Blot <emmanuel.blot@free.fr>
---
 hw/sensor/tmp421.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hw/sensor/tmp421.c b/hw/sensor/tmp421.c
index 59225e8bfd..e92a92d0ba 100644
--- a/hw/sensor/tmp421.c
+++ b/hw/sensor/tmp421.c
@@ -383,6 +383,8 @@ static void tmp421_initfn(Object *obj)
         object_property_add(obj, name, "int",
                             tmp421_get_temperature,
                             tmp421_set_temperature, NULL, NULL);
+        object_property_set_description(obj, name,
+                            "Temperature, in millidegrees Celsius");
     }
 }
 
-- 
2.53.0-Meta

